oracle/tests/test_unit.py

```diff
@@ -137,3 +137,20 @@ def test_handle_query_error_when_connected_disconnects_and_resets_connection(ins
     assert error == 'foo'
     assert oracle_check._cached_connection is None
     cached_connection.assert_has_calls([mock.call.close()])
+
+
+def test_custom_query_metric_prefix_skip_column():
```

Comment:
Looking at this test I don't see anything tested or asserted. Today, looking at the PR I can deduce the purpose... but tomorrow if someone sees this, they probably won't understand the reason for it.
Also, you are testing a private method.

Reply:
I rewrote the test to be a bit more comprehensive and make sure to assert for a metric I mock.